test_data/indent_paren.good.nix

```diff
@@ -0,0 +1,15 @@
+{
+  foo = (
+    (
+      (
+        92
+      )
+    )
+  );
+  xxx = listToAttrs (
+    concatMap (
+      name:
```

Comment:
I've decided to force `\n` between `(` and `name`. We can special case this, and allow to omit linebreak before the lambda